concepts/05 UI Components/Switch/00 Getting Started with Switch/25 Handle Value Change.md

```diff
@@ -0,0 +1,102 @@
+Users can change a component value (*state*) with a simple click. Implement the [onValueChanged](/Documentation/ApiReference/UI_Components/dxSwitch/Configuration/#onValueChanged) callback to handle value changes.
```

Comment:
"with a simple click" - I assume you can also use touchscreens and the switch supports keyboard too (for accessibility). So either we don't specify how to change the value... Or we should list all the ways to change the value. Otherwise, it looks like the component lacks functionality. Or at least I'm starting to think about (does it NOT support keyboard input?).

BTW, I'm not sure about the SPACE key - so please correct my version if I'm wrong. 

```suggestion
Users can change a component value (*state*) in a number of ways: mouse click, SPACE key, or tap. Implement the [onValueChanged](/Documentation/ApiReference/UI_Components/dxSwitch/Configuration/#onValueChanged) callback to handle value changes.
```